app/javascript/packs/project_form_modal.jsx

```diff
@@ -1,44 +1,143 @@
-import React from "react";
+import React from 'react';
+import ReactDOM from "react-dom";
+import { Button, Modal, ModalHeader, ModalBody, ModalFooter, Form, FormGroup, Label, Input } from 'reactstrap';
 import axios from "axios";
-import ReactDOM from "react-dom"
 
 class ProjectFormModal extends React.Component {
-	state = {
-	  title: "",
-		desc: "",
-		key: "",
-	};
+  constructor(props) {
+    super(props);
+    this.state = {
+      modal: false,
+			data: {
```

Comment:
you have very inconsistent indents in your file. Consider converting all tabs to spaces and/or using some code formatter like prettier

Reply:
we use prettier after merge all pr